test/agentchat/test_assistant_agent.py

```diff
@@ -68,6 +68,7 @@ def test_gpt35(human_input_mode="NEVER", max_consecutive_auto_reply=5):
         filter_dict={
             "model": {
                 "gpt-3.5-turbo",
+                "gpt-35-turbo",
```

Comment:
Looks like this might have failed the test?

Reply:
A new deployment with "." in its name cannot be created on Azure, this is added for local testing. It should make no difference for CI.

I'll look into the failing test later tonight.